pom.xml

```diff
@@ -29,8 +29,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.5.1</version>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
This is probably worth making a separate commit.

Reply:
I see, but this pr won't work without java 8

Reply:
Sorry, a separate commit would be fine. Just as long as I can merge it into master separately.